test/internalTest.js

```diff
@@ -473,9 +505,10 @@ for (const supportedVersion of mineflayer.testedVersions) {
           // Versions prior to 1.11 have capital first letter
           const entities = bot.registry.entitiesByName
           const creeperId = entities.creeper ? entities.creeper.id : entities.Creeper.id
-          client.write('spawn_entity_living', {
+          client.write(bot.registry.supportFeature('consolidatedEntitySpawnPacket') ? 'spawn_entity' : 'spawn_entity_living', {
             entityId: 8, // random
             entityUUID: '00112233-4455-6677-8899-aabbccddeeff',
+            objectUUID: '00112233-4455-6677-8899-aabbccddeeff',
```

Comment:
In 1.19, `spawn_entity_living` was merged into `spawn_entity`

`spawn_entity` and `spawn_entity_living` are compatible with each other except for the fact that you need to write both entityUUID and objectUUID. To remove needing to fill out both fields we could replace that parameter with `uuid` on all versions